cli/ops/fetch.rs

```diff
@@ -96,13 +95,10 @@ async fn op_fetch(
     res_headers.push((key.to_string(), val.to_str().unwrap().to_owned()));
   }
 
-  let body = HttpBody::from(res);
-  let rid = state.borrow_mut().resource_table.add(
-    "httpBody",
-    Box::new(StreamResourceHolder::new(StreamResource::HttpBody(
-      Box::new(body),
-    ))),
-  );
```

Comment:
Why not use `HttpBody` type from `http_util.rs`? (or better move that type here)